lib/phoenix_live_view.ex

```diff
@@ -1561,6 +1561,52 @@ defmodule Phoenix.LiveView do
 
         {:ok, socket}
       end
+
+  ### Extracting events for code organization
```

Comment:
I think we should just add a link to the new section here https://github.com/phoenixframework/phoenix_live_view/pull/3685/files#diff-de09b4956b6171d9230ed1417e4546fb5b8b9d7bfd41f9cdf88ee3c21a0138c6R250

As it gives a more complete overview of how this works.